cmd/root.go

```diff
@@ -20,24 +21,33 @@ var (
 
 // global flags
 var (
-	backend string
-	debug   bool
-	version string
+	backend          string
+	debug            bool
+	version          string
+	analyticsEnabled bool
+	analyticsClient  analytics.Client
+	username         string
+)
+
+const (
+	AnalyticsWriteKey = "qK4bR1ogitMozAAFUw3om77inuUjvwbJ"
```

Comment:
left from debugging? not sure if we want this to be in the open source version

Reply:
Write keys are public right?  I mean, we put them in javascript...

Reply:
Yep, they can be - depends on how strict you want to be.